mod/caching/lib/card/cached_count.rb

```diff
@@ -21,8 +21,8 @@ def cached_count
     end
 
     module ClassMethods
-      def expired_cached_count_cards args={}, &block
-        if (set_of_changed_card = args.delete(:set))
+      def recount_trigger set_of_changed_card, args={}, &block
```

Comment:
Assignment Branch Condition size for recount_trigger is too high. [17.52/15]<br>Method has too many lines. [18/15]